src/components/TaskTable.vue

```diff
@@ -27,6 +31,7 @@
           </div>
         </th>
         <th @click="handleFilter('limit')">
```

Comment:
Remain and limit uses the same filter. Order by closest date. I'll rename it, or stick a comment on this.